### DISCUSSION TITLE
Can't see 'components' folder

### DISCUSSION THREAD

**Discussion by blidden** [LekoArts/gatsby-themes#781] [Questions]
The [documentation references](https://github.com/LekoArts/gatsby-starter-minimal-blog#extending-the-footer-of-the-post) a `Post` component - however I don't see a component file or folder anywhere in the project. Am I missing something obvious or do I need to do something to expose it? Thank you :)

**Comment by ch4nd4n** [comment]
You have to manually create it. To do that you can copy over the content from node_modules and then override it.

From my blog(I worked on same thing few days back and blogged about it.)

> To this I figured I have to change the Home layout in Lekoart. 
> I copied homepage.tsx from node_modules/@lekoarts/gatsby-theme-minimal-blog/src/components/ to project src folder.

source: https://www.chandankumar.com/tutorial/gatsby-theme-shadowing-lekoarts
